sum-two-numbers.py

```diff
@@ -0,0 +1,2 @@
+def sum_two_numbers(a,b):
```

Comment:
Can you add documentation to this function?

Reply:
Sure!